template.yaml

```diff
@@ -611,14 +611,34 @@ Resources:
               "Next": "AWSConsoleSignInAlertState",
               "Branches": [
                 {
-                  "StartAt": "CloudtrailActionState",
+                  "StartAt": "CloudtrailChoiceState",
```

Comment:
@mayankbharadwaj  Can you explain these changes , over call?

Reply:
addressed